src/Utilities/FileSystem/FileManager.cpp

```diff
@@ -37,20 +37,21 @@
 namespace MxEngine
 {
 
-    MxString FileManager::OpenFileDialog()
+    MxString FileManager::OpenFileDialog(bool ForMesh)
     {
-        auto selection = pfd::open_file("Select a file", ".",
-            { "Image Files", "*.png *.jpg *.jpeg *.bmp *.tga *.hdr",
-              "All Files", "*" },
-            pfd::opt::multiselect).result();
+        std::vector<std::string> selection;
+        if (ForMesh) {
+            selection = pfd::open_file("Select a file").result();
+        }
+        else {
+            selection = pfd::open_file("Select a file", ".",
+                { "Image Files", "*.png *.jpg *.jpeg *.bmp *.tga *.hdr",
+                  "All Files", "*" },
+                pfd::opt::multiselect).result();
+        }
         return selection.empty() ? "" : ToMxString(selection.front());
 
     }
-    MxString FileManager::OpenFileDialogForMesh()
```

Comment:
I feel like this function was OK, it just should call `OpenFileDialog` with an appropriate selection